src/sentry/ingest/consumer/simple_event.py

```diff
@@ -12,6 +15,13 @@
 logger = logging.getLogger(__name__)
 
 
+consumer_type_to_default_topic = {
+    "events": Topic.INGEST_EVENTS,
+    "transactions": Topic.INGEST_TRANSACTIONS,
+    "attachments": Topic.INGEST_ATTACHMENTS,
+}
+
+
```

Comment:
This does not cover the attachment topic. 
The functions for attachments are `process_attachments_and_events` and `decode_and_process_chunks`. 
I think we should cover all of them. Whether you want to do attachment in a separate PR that's ok.

Reply:
Yes. One step at a time.